src/QueueAnim.jsx

```diff
@@ -169,7 +172,13 @@ class QueueAnim extends React.Component {
   performEnterBegin(key, i) {
     const childrenShow = this.state.childrenShow;
     childrenShow[key] = true;
-    this.setState({childrenShow}, this.realPerformEnter.bind(this, key, i));
+    /*
+     *这里会报错,用_reactInternalInstance来分别出这个组件还存在否,
+     * componentWillUnmount能去掉所有的setState吗?这里有延时处理...
+     */
+    if (this._reactInternalInstance) {
```

Comment:
6没了吧。。。
 isMounted(...) is deprecated in plain JavaScript React classes. Instead, make sure to clean up subscriptions and pending requests in componentWillUnmount to prevent memory leaks.